--- /lib/systemd/system/ospi.service.orig	2015-07-01 09:00:02.086984370 +0000
+++ /lib/systemd/system/ospi.service	2015-07-01 09:02:38.056984389 +0000
@@ -2,8 +2,8 @@
 Description=Start Python Interval program
 
 [Service]
-WorkingDirectory=/home/root/ospi
-ExecStart=/usr/bin/python ospi.py
+WorkingDirectory=/usr/share/pyshared/ospi/
+ExecStart=/usr/bin/python __init__.py
 KillMode=process
 
 [Install]
